components/ProfileCard.vue

```diff
@@ -0,0 +1,55 @@
+<script setup>
+
+	const props = defineProps({
+		firstName: {
+			type: String,
+			required: true,
+		},
+		lastName: {
+			type: String,
+			required: true,
+		},
+		functionTitle: {
```

Comment:
Free info: the default value for `required` is `false` so you don't have to set is explicitly.